dispatcher/Node.go

```diff
@@ -78,30 +78,36 @@ type Node struct {
 	peerAddr      *net.UDPAddr
 	failureCtrl   *FailureCtrl
 	isFailed      bool
-	radioRange    int
+	radioRange    float64
+	radioRangeViz int
```

Comment:
Currentl they can be independently set. radioRange is the max possible radio range in meter (but it might be less than this, due to the RSSI getting very low already before radioRange is reached.)

radioRangeViz is the number of pixels that the animated radio range is showing on screen. And it is also used to set radio range by the Python-driven scripts.  For this reason the code at the moment keeps both in sync in order not to disturb the scripted tests.

There's a hardcoded conversion of 1 pixel == 0.10 m distance.

Reply:
The latest commit brings the conversion factor into the Simulation configuration. (Value gets then copied into Dispatcher configuration object.) The "radioRangeViz" is changed back to "radioRange" to keep things simple. The visualization currently will always visualize the max radio range, even if the max cannot be always reached -- like in an RSSI based model.  But by default (range=160) an RSSI based model will always reach the end of transmission range easily.

Reply:
Generally speaking, dispatcher should not care about visualizing. Visualizers are responsible for visualization.

Reply:
Fixed now in latest commits - "radiorange" is now universally used. For the RSSI-based RF models it represents the max range at which link quality 1 is still achievable.  So the conversion pixels/meters is not needed anymore in most places, and the conversion is implicit and confined to the radiomodel package only.